### ISSUE TITLE
Missing Item - Warning regarding Wakeners Frond

### ISSUE THREAD

**Issue by Baine** [Hekili/hekili#706]
**Describe the Bug/Issue**
Warning Message in Option Interface Regarding Wakeners Frond

**To Reproduce**
- Have Wakeners Frond (https://www.wowhead.com/item=181457) with iLvl 161 equipped
- Open Hekili Options Interface
- Go to Warnings
- See Entry

```
Unable to find wakeners_frond / 181457 / unknown in the itemlist.

[string "@Interface\AddOns\Hekili\Options.lua"]:3929: in function `EmbedItemOption'
[string "@Interface\AddOns\Hekili\Classes.lua"]:550: in function `func'
[string "@Interface\AddOns\Hekili\Events.lua"]:218: in function `handler'
[string "@Interface\AddOns\Hekili\Events.lua"]:56: in function <Interface\AddOns\Hekili\Events.lua:50>
```

**Expected behavior**
no warning

**WHEN SUBMITTING AN ISSUE:**
1.  The information from the Issue Reporting tab ( /hekili > Issue Reporting ).  This provides your talent and gear information.  You can paste it to pastebin.com and provide a link here.

https://pastebin.com/iBsLV1eY

**Comment by Hekili** [comment]
Warnings are fine, but I appreciate the report.

**Comment by Baine** [comment]
I also don't see it in the suggested actions. got a darkmoon card which is suggested on cooldown, but not this one.

just an fyi :) if it's a "will not fix" type of warning this can be closed

**Comment by Hekili** [comment]
If it's not being recommended, you'll need to provide a snapshot taken when you thought it should've been recommended, but wasn't.

2.  If reporting a disagreement with the addon's recommendations, please take a snapshot when you observe the issue and provide the snapshot.  This is not a screenshot.  A snapshot can be generated by binding the Pause feature on the Toggles tab.  When you press Pause, a snapshot of how the addon decided on its CURRENT recommendations is generated and saved to the Snapshots tab.  Copy and paste this to pastebin.com and provide the link here.
 
**PUT SNAPSHOT LINK HERE:**

**Comment by Baine** [comment]
my apologies, assumed that it's not included in priorities due to the warning.
Snapshot Link suggesting Darkmoon Card instead of Frond:
https://pastebin.com/zeiWhAxk

also, since I can't find the proper item lvl for this trinket on wowhead, this is the simc string for this trinket:
trinket1=,id=181457,bonus_id=6652/1472/5858/6616

**Comment by Prejudice182** [comment]
This is the dump from Wakener's Frond. I am thinking of just adding it to the cooldowns priority list, like other trinkets.

[20:54:20] Dump: value=Hekili.Class.abilities.wakeners_frond 
[20:54:20] [1]={ 
[20:54:20]   gcd="off", 
[20:54:20]   itemSpellName="Wakener's Frond", 
[20:54:20]   realCast=0, 
[20:54:20]   auras={ 
[20:54:20]     wakeners_frond={ 
[20:54:20]       id=336588, 
[20:54:20]       duration=12, 
[20:54:20]       max_stack=1 
[20:54:20]     } 
[20:54:20]   }, 
[20:54:20]   key="wakeners_frond", 
[20:54:20]   toggle="cooldowns", 
[20:54:20]   texture=3536178, 
[20:54:20]   itemLoaded=26812.16, 
[20:54:20]   charges=1, 
[20:54:20]   recharge=120, 
[20:54:20]   link="|cff1eff00|Hitem:183850::::::::::::::::|h[Wakener's Frond]|h|r", 
[20:54:20]   id=-228, 
[20:54:20]   itemSpellID=336588, 
[20:54:20]   funcs={ 
[20:54:20]     cast=<function>, 
[20:54:20]     item=<function> 
[20:54:20]   }, 
[20:54:20]   name="Wakener's Frond", 
[20:54:20]   itemSpellKey="wakeners_frond_336588", 
[20:54:20]   cooldown=120, 
[20:54:20]   handler=<function> 
[20:54:20] }

**Comment by Baine** [comment]
just noticed that this dump has a different itemid than trinket actually has (at least in the link field)

**Comment by molmr** [comment]
I am also experiencing the same issue, Hekili never recommends Wakener's Frond despite me wearing it. Please see my pastebin: https://pastebin.com/vnASUbL2

**Comment by molmr** [comment]
9.0.2-1.0.25b seems to have fixed this, thank you!

**Comment by Baine** [comment]
fixed since 0d7d8dc06ee65649316624ade5a8c6468928492e

thanks <3
I'll close this one

**Comment by Hekili** [comment]
Thank you both for confirming.
